inc/deprecated/3.6.php

```diff
@@ -0,0 +1,14 @@
+<?php
+// phpcs:ignoreFile
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class aliases.
+ */
+class_alias( '\WP_Rocket\ThirdParty\Plugins\Smush', '\WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber' );
+class_alias( '\WP_Rocket\Admin\Settings\Page', '\WP_Rocket\Engine\Admin\Settings\Page' );
+class_alias( '\WP_Rocket\Admin\Settings\Render', '\WP_Rocket\Engine\Admin\Settings\Render' );
+class_alias( '\WP_Rocket\Admin\Settings\Settings', '\WP_Rocket\Engine\Admin\Settings\Settings' );
+class_alias( '\WP_Rocket\ServiceProvider\Settings', '\WP_Rocket\Engine\Admin\Settings\ServiceProvider' );
+class_alias( '\WP_Rocket\Subscriber\Admin\Settings\Page_Subscriber', '\WP_Rocket\Engine\Admin\Settings\Subscriber' );
```

Comment:
- since it's a PR going into a minor 3.5 version, it should be in the 3.5 deprecated file, not 3.6
- Shouldn't it use a double `\\` instead of a simple? Or are both fine?

Reply:
Darn it. Totally missed that it's based to `develop`. Doh. I'll fix it up.

A simple `\` is all that's needed. Think of how we do it in the ServiceProviders. So to answer your question: both work.

Reply:
Fixed in 577a1bda3500efc262104bc024abf99c76f543b0.